scripts/core/editor3/components/links/LinkInput.tsx

```diff
@@ -80,13 +82,19 @@ export class LinkInputComponent extends React.Component<any, any> {
      */
     onSubmit(linkType) {
         let link;
-        const {suggestingMode} = this.props;
+        const {suggestingMode, localDomains} = this.props;
         const _createLinkSuggestion = this.props.createLinkSuggestion;
         const _applyLink = this.props.applyLink;
         const _changeLinkSuggestion = this.props.changeLinkSuggestion;
 
         if (linkType === linkTypes.href) {
-            link = {href: this.state.url};
+            const url = this.state.url;
+            const isLocalDomain = (localDomains || []).some((item) => url.includes(item.domain));
+
+            link = {
+                href: url,
+                target: isLocalDomain ? '_self' : '_blank',
```

Comment:
maybe it could be just `null` instead of `'_self'` it's not needed to be there

Reply:
I've just never seen it used anywhere, it's the default value after all.
and it changes the output, for that reason I would avoid it.

Reply:
I changed the code to don't set any target if local_domains cv is not defined.
Also the target is set only for external links when local_domains cv is defined.